src/frontend/src/optimizer/plan_node/stream_table_scan.rs

```diff
@@ -233,6 +240,32 @@ impl StreamTableScan {
             })
             .collect_vec();
 
+        // TODO: snapshot read of upstream mview
```

Comment:
@StrikeW I saw this TODO added by you previously. I guess we want to provide a common interface for `snapshot_read`, under `backfill::upstream_table::snapshot`?